File: DataAccessLayer/Models/Vehicle.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DataAccessLayer.Models
{
    public class Vehicle
    {
        public int Id { get; set; }
        public string Make { get; set; }
        public string Model { get; set; }
        public int YearOfManufacture { get; set; }
        public int InitialMileage { get; set; }
        public bool Available { get; set; }

        public override string ToString()
        {
            return $"{Id} - {Make} {Model}";
        }

    }
}

File: bulk-import-module/src/hr/mucnjakf/models/Vehicle.java
package hr.mucnjakf.models;

public class Vehicle {

    private String make;
    private String model;
    private int yearOfManufacture;
    private int initialMileage;
    private boolean available;

    public Vehicle(String make, String model, int yearOfManufacture, int initialMileage, boolean available) {
        this.make = make;
        this.model = model;
        this.yearOfManufacture = yearOfManufacture;
        this.initialMileage = initialMileage;
        this.available = available;
    }

    public String getMake() {
        return make;
    }

    public String getModel() {
        return model;
    }

    public int getYearOfManufacture() {
        return yearOfManufacture;
    }

    public int getInitialMileage() {
        return initialMileage;
    }

    public boolean isAvailable() {
        return available;
    }

    @Override
    public String toString() {
        return "Vehicle{" +
                "make='" + make + '\'' +
                ", model='" + model + '\'' +
                ", yearOfManufacture=" + yearOfManufacture +
                ", initialMileage=" + initialMileage +
                ", available=" + available +
                '}';
    }
}
